cmake/syscheck.cmake

```diff
@@ -7,7 +7,8 @@ if (WITH_ZEPHYR)
   if (NOT WITH_ZEPHYR_LIB)
     include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
   endif()
-  if (CONFIG_CPU_CORTEX_M)
-    set (MACHINE "cortexm" CACHE STRING "")
-  endif (CONFIG_CPU_CORTEX_M)
+  if (CONFIG_ARM)
```

Comment:
That's correct, since ARM signifies, actually AARC32. So it does not enable AARC64. 
That's just an FYI, and perhaps we need to change the PR title/body.

Reply:
Thanks, it's done !